tests/tracking/integration_test_utils.py

```diff
@@ -13,7 +13,7 @@
 _logger = logging.getLogger(__name__)
 
 
-def _await_server_up_or_die(port, timeout=20):
+def _await_server_up_or_die(port, timeout=60):
```

Comment:
```suggestion
def _await_server_up_or_die(port, timeout=30):
```

Reply:
We can increase timetout again if 30 is insufficient.

Reply:
okay